core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -231,6 +231,10 @@ private TableProperties() {}
   public static final String ORC_BATCH_SIZE = "read.orc.vectorization.batch-size";
   public static final int ORC_BATCH_SIZE_DEFAULT = 5000;
 
+  public static final String DELETE_COLUMN_STATS_FILTERING_ENABLED =
+      "read.delete.column-stats-filtering.enabled";
+  public static final boolean DEFAULT_DELETE_COLUMN_STATS_FILTERING_ENABLED = true;
```

Comment:
To keep the name consistent with other properties. It would be better called `DELETE_COLUMN_STATS_FILTERING_ENABLED_DEFAULT`

Reply:
Oh, yeah, I don’t know why I named it that way. I will fix it.